homeassistant/components/tami4/coordinator.py

```diff
@@ -0,0 +1,61 @@
+"""Water quality coordinator for Tami4Edge."""
+from dataclasses import dataclass
+from datetime import date, timedelta
+import logging
+
+from Tami4EdgeAPI import Tami4EdgeAPI, exceptions as APIExceptions
```

Comment:
```suggestion
from Tami4EdgeAPI import Tami4EdgeAPI, exceptions
```

So it's easier to understand that `exceptions` is a module